### PR TITLE
[INJICERT-695] add Mock mdoc MDL config

### PR DESCRIPTION
**Pull request by KiruthikaJeyashankar**

### CODE REVIEW

File: certify-mock-mdl.properties [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+mosip.certify.integration.audit-plugin=LoggerAuditService
+mosip.certify.integration.vci-plugin=MDocMockVCIssuancePlugin
+mosip.certify.issuer=PluginIssuer
+
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+mosip.certify.cache.secure.individual-id=false
+mosip.certify.cache.store.individual-id=true
+mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
+mosip.certify.mock.vciplugin.verification-method=${mosip.certify.authn.jwk-set-uri}
+mosip.certify.mock.authenticator.get-identity-url=https://${mosip.api.public.host}/v1/mock-identity-system/identity
```
[the remark this answers is not in the document]
Comment (KiruthikaJeyashankar) [reply certify-mock-mdl.properties:16]: Yes we are not needed with identity-url here as we are not reading VC data from mock DB
Will remove unused props as per mdoc usecase

[the remark this answers is not in the document]
Comment (KiruthikaJeyashankar) [reply certify-mock-mdl.properties]: modified

File: certify-mock-mdl.properties [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+mosip.certify.integration.scan-base-package=io.mosip.certify.mock.integration
+mosip.certify.integration.audit-plugin=LoggerAuditService
+mosip.certify.integration.vci-plugin=MDocMockVCIssuancePlugin
+mosip.certify.plugin-mode=VCIssuance
+
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+mosip.certify.cache.secure.individual-id=false
+mosip.certify.cache.store.individual-id=true
+mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
+mosip.certify.mock.vciplugin.mdoc.issuer-key-cert=${mosip.certify.mock.vciplugin.mdoc.issuer-key-cert}
```
Comment (vishwa-vyom) [inline certify-mock-mdl.properties:15]: why do we have both key and value same ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: certify-mock-mdl.properties (hunk 2 above)
Comment (KiruthikaJeyashankar) [reply certify-mock-mdl.properties:15]: Changed the placeholder value as well to reflect the property key change
@vishwa-vyom Can we keep the placeholder as `mosip.certify.mock.vciplugin.mdoc.issuer-crypto-details` or as previous `mosip.certify.mock.vciplugin.issuer.cert`

Comment (KiruthikaJeyashankar) [reply certify-mock-mdl.properties]: Modifying the value to mosip.certify.mock.mdoc.issuer-key-cert

**Review by vishwa-vyom — APPROVED** [review]

**Review by swatigoel — APPROVED** [review]

**Pull request merged**